### PR TITLE
feat: added user inputs and Weather API

### PR DESCRIPTION
**Pull request by znjs**
- Added user name input
- Added focus input
- fetching weather from open weather

### CODE REVIEW

**Review by ScoobyDubeyDo — APPROVED** [review]
The code looks clean
Good naming conventions

**Review by muskaanshah — APPROVED** [review]
Code is clean, good to merge

**Comment by muskaanshah** [comment]
Code is clean, good to merge

**Comment by muskaanshah** [comment]
Code is clean, good to merge

File: src/pages/home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+const TIME = tw.time`text-9xl`;
+const TIMEWRAPPER = tw.div`flex flex-row items-center select-none`;
+const QUOTE = tw.q`absolute bottom-0 right-0 left-0 mx-auto text-center text-xl cursor-default`;
+const SEARCH = tw.input` text-gray-900 outline-none bg-transparent border-b-2 border-black search text-center`;
+const SEARCHWRAPPER = tw.div`p-4 relative z-10 flex items-center`;
+const SEARCHICON = tw.i`text-gray-900 text-2xl fa-solid fa-magnifying-glass cursor-pointer`;
+const FLEXROWCONTAINER = tw.div`flex items-center justify-center gap-x-4`;
+const FOCUSEDIT = tw.i`fa-solid fa-pencil transition cubic-bezier(0.38, 0.25, 0.42, 0.71) delay-400 hover:bg-[#8080809c] cursor-pointer  p-3 rounded-full`;
+const USEREDIT = tw.i`fa-solid fa-pencil transition cubic-bezier(0.38, 0.25, 0.42, 0.71) delay-400 hover:bg-[#8080809c] cursor-pointer p-3 rounded-full`;
+const FOCUSCHECK = tw.i`fa-solid fa-check transition cubic-bezier(0.38, 0.25, 0.42, 0.71) delay-400 hover:bg-[#8080809c] cursor-pointer p-3 rounded-full`;
```
Comment (AnjaliDeshwani) [inline src/pages/home/Home.jsx:177]: Liked the way you have wrapped these elements. Got to know new thing.

**Review by AnjaliDeshwani — APPROVED** [review]
Looks good to merge.

**Pull request merged**